src/app/base/components/PageContent/PageContent.test.tsx

```diff
@@ -29,3 +38,79 @@ it("displays hidden sidebar when no content provided", () => {
   );
   expect(screen.queryByRole("complementary")).toHaveClass("is-collapsed");
 });
+
+it("shows the secondary navigation for settings", () => {
+  state.status.authenticated = true;
+  state.status.connected = true;
+  renderWithBrowserRouter(
+    <PageContent
+      header="Settings"
+      sidePanelContent={null}
+      sidePanelTitle={null}
+    >
+      content
+    </PageContent>,
+    { route: "/settings/configuration/general", state }
+  );
+
+  expect(screen.getByRole("navigation")).toBeInTheDocument();
+
+  settingsNavItems.forEach((item) => {
+    expect(screen.getByText(item.label)).toBeInTheDocument();
+  });
+});
+
+it("shows the secondary navigation for preferences", () => {
+  state.status.authenticated = true;
+  state.status.connected = true;
+  renderWithBrowserRouter(
+    <PageContent
+      header="Preferences"
+      sidePanelContent={null}
+      sidePanelTitle={null}
+    >
+      content
+    </PageContent>,
+    { route: "/account/prefs/details", state }
+  );
```

Comment:
This is fine as is, but generally I'd be for merging these 2 tests into 1. 
https://kentcdodds.com/blog/write-fewer-longer-tests